getting-started/01-overview.story.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import './css/global.css';
+import logo from './assets/rainbow-logo.svg';
+import puzzle from './assets/puzzle.svg';
+import accesibility from './assets/accesibility.svg';
+import translation from './assets/translation.svg';
```

Comment:
can we use react components instead of using the <img /> tag?

Reply:
you mean for images?